src/Features/Lsif/Generator/Microsoft.CodeAnalysis.LanguageServerIndexFormat.Generator.csproj

```diff
@@ -41,9 +41,12 @@
     </ItemGroup>
   </Target>
 
+  <ItemGroup>
+    <Compile Include="..\..\LanguageServer\Protocol\Handler\FoldingRanges\FoldingRangesHelper.cs" Link="FoldingRangesHelper.cs" />
```

Comment:
I felt linking the folding range helper directly into the `LsifGenerator` project was better then adding an IVT from `Microsoft.VisualStudio.LanguageServer.Protocol` to `LsifGenerator` project. Let me know if you feel otherwise.

Reply:
IVT from LanguageServer.Protocol to LsifGenerator is 100% expected in my book -- indeed that was one of the reasons we moved it into this repo was to take advantage of IVT without risks. The only reason this wasn't already there was by pure chance the features we implemented in LSIF didn't actually need the code from LSP. In this case, IVT is good.